balancer-labs/frontend-v2
# UI/apyvision

# Description
UI clean up of new APY.vision link on Pool pages.

## Type of change

- [x] Other: UI clean up of an existing PR

## How should this be tested?

Go to any pool page and view the link under the pool stats section. It should look like this:

<img width="1063" alt="Screen Shot 2022-04-04 at 3 12 59 PM" src="https://user-images.githubusercontent.com/1121139/161562959-c6c995cf-698d-4d55-9cf6-1182cccc78e6.png">
